assets/js/googlesitekit/modules/datastore/modules.test.js

```diff
@@ -143,13 +134,7 @@ describe( 'core/modules modules', () => {
 				);
 
 				muteConsole( 'error' );
-				registry.dispatch( STORE_NAME ).activateModule( slug );
-
-				// Wait until this activation action has completed.
-				await subscribeUntil( registry, () => registry
-					.select( STORE_NAME )
-					.isDoingSetModuleActivation( slug ) === false
-				);
+				await registry.dispatch( STORE_NAME ).activateModule( slug );
```

Comment:
See above.

Reply:
Yep. Even though this action doesn't have a control of its own, we can still await it to wait for all of its internally `yield`ed actions (async or not) to be done.